lib/resty/dns/resolver.lua

```diff
@@ -81,6 +83,7 @@ local resolver_errstrs = {
     "refused",          -- 5
 }
 
+local soa_names = { "serial", "refresh", "retry", "expire", "mininum"}
```

Comment:
You are using a space after `{`, but there is no space before `}`. This is unbalanced.